### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by rkapoor10**
docs
    ├── clusters                  
    ├── connectors             
    ├── delegates                   
    ├── secrets                   
    ├── GettingStarted.md                # this is the starting point
    └── PreFlightChecklist.md

### CODE REVIEW

**Review by hrittikhere — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +0,0 @@
-# cd-pipeline-sample
```
Comment (hrittikhere) [inline README.md:1]: Create a TOC

Comment (hrittikhere) [reply README.md:1]: Table of contents

File: docs/secrets/Introduction.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+
+# SECRETS
+
+
+Harness uses **secrets** to securely store things like access **tokens**. 
+  
+We’re going to walk you through creating the GitHub secret. Then you’ll have a choose-your-own-adventure path for the remaining secrets. You know, the ones for Docker Hub
```
Comment (hrittikhere) [inline docs/secrets/Introduction.md:7]: End with an exclamation mark

**Review by hrittikhere — CHANGES_REQUESTED** [review]

File: docs/GettingStarted.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# Getting Started
+
+> Before you start make sure to follow this workflow for a smooth onboarding experience.  
+`~ Captain Canary `
+
+```mermaid
+graph LR
+
+PreFlightChecklist --> Secrets --> Clusters --> Delegates --> Connectors
```
[the remark this answers is not in the document]
Comment (rkapoor10) [reply docs/GettingStarted.md:9]: done

File: docs/PreFlightChecklist.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# Pre-flight Checklist
+
+Make sure you tick all the check boxes before you begin you CD Journey 🚀
+
+- [ ] **Forking the Repository** - Fork this repository on your github: [YAML-Onboarding Repo](https://github.com/Debanitrkl/YAML-Onboarding-Harness)
+
+- [ ] **A Version Control System account**  - In this guide we’re going to use GitHub. Our example app’s source is on GitHub and you’ll need to fork those repos, so make sure you have a GitHub account. Alternatively, if you’re comfortable following your inner-shoulder-angel’s guide to converting GitHub instructions you can use your own VCS. This, however, is an exercise for the reader.
```
[the remark this answers is not in the document]
Comment (rkapoor10) [reply docs/PreFlightChecklist.md:7]: done

File: docs/delegates/Delegate.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
+# Harness Delegate
+
+Our delegate serves two purposes. First, to provide a secure communication link between **Harness** and your **computer environment**. Additionally, it allows Harness to delegate CI/CD tasks to your compute environment. Yeah, it’s why we named it the **delegate**. #NamingThingsIsHard.
```
[the remark this answers is not in the document]
Comment (rkapoor10) [reply docs/delegates/Delegate.md:3]: replaced with GKE cluster.

File: docs/secrets/Introduction.md (hunk 2 above)
Comment (rkapoor10) [reply docs/secrets/Introduction.md:7]: done

File: README.md (hunk 1 above)
Comment (rkapoor10) [reply README.md:1]: added

**Comment by hrittikhere** [comment]
Changes Approved!

**Review by hrittikhere — APPROVED** [review]
LGTM!

**Review by Ompragash — APPROVED** [review]

**Pull request merged**